client/src/app/site/pages/meetings/pages/agenda/modules/topics/modules/topic-poll/components/topic-poll-meta-info/topic-poll-meta-info.component.html

```diff
@@ -23,8 +23,8 @@
             {{ 'Entitled to vote' | translate }}:
 
             <span *ngFor="let group of poll.entitled_groups; let last = last">
-                {{ group.getTitle() | translate }}
-                <span *ngIf="!last">,</span>
+                <span>{{ group.getTitle() | translate }}</span>
+                <span>{{ !last ? ', ' : '' }}</span>
```

Comment:
I think this would be better. 
```suggestion
                <span>{{ group.getTitle() | translate }}</span>
                <span *ngIf="!last">,</span>
```